nuxt.config.js

```diff
@@ -141,7 +141,7 @@ export default {
 
   devServer: {
     proxy: {
-      // this is needed to prevent a CORS error when running locally (will need to update with url)
+      // this is needed to prevent a CORS error when running locally
```

Comment:
It's working atm, but I haven't figured out why 3 `keycloak.json` files still need to exist. I'll work on this in [# 10697](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10697).